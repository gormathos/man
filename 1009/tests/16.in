sol_id;user_id;task_id;score;posted_time
147160;1088;2007;100.00;1692954063
147161;1097;2009;67.65;1692955432
147163;1005;2002;19.05;1692960003
147166;1069;2001;100.00;1692964086
147170;1067;2002;57.14;1692967434
147175;1038;2004;86.49;1692972580
147181;1058;2005;55.26;1692976825
147188;1098;2007;80.65;1692977673
147196;1057;2001;2.08;1692978846
147205;1008;2003;40.00;1692984471
147215;1037;2010;100.00;1692986235
147226;1094;2006;100.00;1692991493
147238;1039;2006;52.08;1692996566
147251;1019;2004;35.14;1692998280
147265;1034;2001;14.58;1692998591
147280;1023;2003;20.00;1693000822
147296;1073;2005;78.95;1693004595
147313;1029;2001;62.50;1693005898
147331;1028;2005;55.26;1693010076
147350;1030;2006;95.83;1693010651
147370;1005;2003;88.00;1693013356
147391;1040;2003;72.00;1693019076
147413;1065;2010;54.55;1693024077
147436;1086;2010;100.00;1693025026
147460;1082;2007;93.55;1693029634
147485;1086;2004;83.78;1693032473
147511;1031;2006;12.50;1693036050
147538;1039;2006;79.17;1693041085
147566;1098;2007;100.00;1693043025
147595;1009;2002;66.67;1693043947
147625;1005;2005;0.00;1693049290
147656;1029;2004;29.73;1693053802
147688;1065;2007;22.58;1693056650
147721;1088;2004;83.78;1693059384
147755;1091;2001;66.67;1693061428
147790;1074;2007;100.00;1693065796
147826;1027;2008;97.78;1693066899
147863;1036;2004;100.00;1693068939
147901;1094;2006;68.75;1693071496
147940;1020;2008;28.89;1693071774
147980;1090;2004;45.95;1693074579
148021;1060;2002;61.90;1693079898
148063;1056;2003;24.00;1693085397
148106;1073;2002;100.00;1693089492
148150;1030;2001;10.42;1693090249
148195;1067;2010;13.64;1693095511
148241;1098;2006;10.42;1693098693
148288;1017;2009;5.88;1693104311
148336;1087;2006;75.00;1693105376
148385;1027;2001;81.25;1693108465
148435;1001;2004;100.00;1693110534
148486;1023;2008;0.00;1693112128
148538;1002;2004;37.84;1693115557
148591;1076;2006;10.42;1693117950
148645;1084;2009;100.00;1693122223
148700;1056;2002;71.43;1693126454
148756;1064;2009;8.82;1693129448
148813;1033;2006;91.67;1693134653
148871;1065;2004;64.86;1693139064
148930;1066;2003;28.00;1693144004
148990;1086;2007;100.00;1693149075
149051;1075;2001;83.33;1693150355
149113;1096;2002;90.48;1693154879
149176;1083;2002;38.10;1693155608
149240;1099;2004;43.24;1693161128
149305;1024;2010;18.18;1693166584
149371;1011;2007;93.55;1693168015
149438;1052;2008;100.00;1693172836
149506;1038;2001;100.00;1693178612
149575;1002;2003;36.00;1693179708
149645;1068;2009;5.88;1693184912
149716;1062;2001;8.33;1693189853
149788;1093;2010;100.00;1693191144
149861;1053;2010;100.00;1693196022
149935;1010;2008;100.00;1693197653
150010;1024;2003;100.00;1693198100
150086;1023;2003;80.00;1693202708
150163;1069;2008;100.00;1693207510
150241;1040;2009;100.00;1693208046
150320;1063;2004;18.92;1693212647
150400;1019;2005;5.26;1693216075
150481;1085;2004;37.84;1693219097
150563;1092;2010;63.64;1693222565
150646;1085;2005;21.05;1693226359
150730;1002;2002;28.57;1693228265
150815;1024;2006;100.00;1693231080
150901;1005;2006;100.00;1693231656
150988;1082;2008;17.78;1693235777
151076;1037;2008;93.33;1693237100
151165;1064;2006;100.00;1693241282
151255;1042;2001;60.42;1693245158
151346;1042;2010;100.00;1693247713
151438;1024;2008;91.11;1693248336
151531;1091;2010;100.00;1693249331
151625;1049;2009;73.53;1693252883
151720;1033;2009;91.18;1693255224
151816;1073;2001;100.00;1693258404
151913;1021;2009;100.00;1693263951
152011;1093;2006;75.00;1693265941
152110;1037;2006;100.00;1693271687
152210;1055;2007;96.77;1693275031
152311;1028;2007;96.77;1693276371
152413;1100;2010;0.00;1693280942
152516;1008;2002;100.00;1693283221
152620;1023;2008;33.33;1693283880
152725;1075;2008;88.89;1693289146
152831;1066;2004;100.00;1693289499
152938;1060;2010;36.36;1693292638
153046;1022;2008;48.89;1693298255
153155;1090;2006;22.92;1693299913
153265;1024;2006;22.92;1693300248
153376;1041;2008;46.67;1693303869
153488;1065;2010;0.00;1693304238
153601;1059;2002;90.48;1693309741
153715;1066;2001;100.00;1693310738
153830;1086;2005;13.16;1693312212
153946;1008;2002;33.33;1693314316
154063;1049;2004;100.00;1693319078
154181;1037;2008;68.89;1693322145
154300;1076;2007;6.45;1693327271
154420;1047;2005;100.00;1693327980
154541;1056;2009;100.00;1693329768
154663;1077;2005;100.00;1693330946
154786;1054;2006;100.00;1693336610
154910;1097;2007;80.65;1693337025
155035;1062;2002;100.00;1693341064
155161;1036;2009;73.53;1693346845
155288;1039;2008;15.56;1693351888
155416;1052;2002;100.00;1693354033
155545;1069;2009;100.00;1693356326
155675;1002;2008;80.00;1693356456
155806;1067;2006;37.50;1693358123
155938;1090;2009;100.00;1693363266
156071;1035;2009;29.41;1693364689
156205;1071;2006;39.58;1693366142
156340;1098;2006;66.67;1693371456
156476;1039;2010;100.00;1693375578
156613;1092;2003;100.00;1693380010
156751;1028;2002;57.14;1693380709
156890;1076;2010;72.73;1693383360
157030;1004;2007;93.55;1693384638
157171;1063;2009;100.00;1693386871
157313;1090;2008;100.00;1693391396
157456;1067;2003;80.00;1693394630
157600;1057;2001;8.33;1693397012
157745;1038;2003;100.00;1693398609
157891;1093;2009;29.41;1693400930
158038;1052;2006;50.00;1693401768
158186;1084;2004;59.46;1693402970
158335;1098;2005;100.00;1693404771
158485;1043;2005;97.37;1693404983
158636;1027;2002;100.00;1693406352
158788;1036;2005;100.00;1693406446
158941;1084;2001;100.00;1693406817
159095;1094;2009;67.65;1693407205
159250;1094;2007;32.26;1693407346
159406;1075;2009;52.94;1693407464
159563;1054;2004;78.38;1693409846
159721;1010;2010;68.18;1693409924
159880;1002;2004;83.78;1693413118
160040;1093;2005;26.32;1693415201
160201;1018;2007;100.00;1693415413
160363;1044;2007;77.42;1693418748
160526;1045;2010;9.09;1693423856
160690;1027;2010;100.00;1693427004
160855;1033;2006;18.75;1693430580
161021;1036;2005;26.32;1693430908
161188;1025;2002;52.38;1693435778
161356;1029;2005;84.21;1693440087
161525;1034;2003;28.00;1693440634
161695;1052;2004;89.19;1693446042
161866;1028;2009;58.82;1693447478
162038;1092;2007;25.81;1693452973
162211;1029;2002;19.05;1693458281
162385;1039;2001;0.00;1693462981
162560;1090;2009;88.24;1693463748
162736;1056;2004;75.68;1693467513
162913;1049;2001;93.75;1693467965
163091;1051;2006;62.50;1693468890
163270;1003;2002;100.00;1693473563
163450;1022;2003;68.00;1693479056
163631;1066;2003;100.00;1693480288
163813;1071;2001;8.33;1693481109
163996;1010;2009;47.06;1693483658
164180;1072;2001;72.92;1693488416
164365;1004;2004;18.92;1693491850
164551;1094;2006;100.00;1693497017
164738;1030;2009;97.06;1693497983
164926;1056;2006;77.08;1693501012
165115;1054;2004;18.92;1693502121
165305;1042;2004;100.00;1693502163
165496;1097;2001;41.67;1693502305
165688;1061;2007;100.00;1693507122
165881;1059;2006;60.42;1693510382
166075;1021;2007;48.39;1693511180
166270;1044;2002;100.00;1693512333
166466;1098;2006;79.17;1693518078
166663;1008;2002;14.29;1693520957
166861;1068;2007;90.32;1693521801
167060;1080;2002;100.00;1693524838
167260;1037;2007;100.00;1693529523
167461;1024;2005;100.00;1693529742
167663;1071;2005;100.00;1693535197
167866;1077;2006;100.00;1693539786
168070;1002;2008;20.00;1693540521
168275;1079;2001;100.00;1693545962
168481;1062;2003;88.00;1693548416
168688;1053;2002;0.00;1693550545
168896;1097;2008;44.44;1693552626
169105;1068;2010;13.64;1693556358
169315;1051;2010;4.55;1693562127
169526;1053;2001;77.08;1693563324
169738;1099;2001;100.00;1693564067
169951;1044;2008;55.56;1693566820
170165;1038;2002;61.90;1693566898
170380;1085;2006;35.42;1693569234
170596;1069;2002;76.19;1693572433
170813;1048;2005;10.53;1693574097
171031;1072;2008;4.44;1693578122
171250;1083;2003;100.00;1693580542
171470;1048;2010;68.18;1693580864
171691;1002;2008;97.78;1693586005
171913;1036;2005;100.00;1693588450
172136;1060;2009;100.00;1693588633
172360;1035;2006;27.08;1693589256
172585;1024;2010;50.00;1693589262
172811;1053;2002;19.05;1693591293
173038;1031;2003;28.00;1693591364
173266;1028;2005;10.53;1693591659
173495;1020;2004;100.00;1693596792
173725;1057;2008;86.67;1693602418
173956;1047;2006;64.58;1693602447
174188;1008;2004;18.92;1693605730
174421;1026;2009;47.06;1693607909
174655;1016;2010;100.00;1693608905
174890;1053;2009;0.00;1693614272
175126;1054;2002;14.29;1693617869
175363;1018;2007;100.00;1693620576
175601;1099;2002;28.57;1693620754
175840;1044;2003;12.00;1693621348
176080;1048;2008;40.00;1693625012
176321;1075;2006;100.00;1693626564
176563;1042;2005;55.26;1693631735
176806;1036;2005;63.16;1693637692
177050;1022;2010;72.73;1693641896
177295;1049;2007;93.55;1693644359
177541;1082;2008;62.22;1693645947
177788;1076;2009;41.18;1693648898
178036;1077;2003;8.00;1693654742
178285;1068;2002;90.48;1693659729
178535;1012;2001;52.08;1693659955
178786;1028;2001;22.92;1693662926
179038;1021;2008;100.00;1693667118
179291;1094;2005;2.63;1693671499
179545;1092;2001;52.08;1693672239
179800;1055;2007;38.71;1693674006
180056;1071;2009;100.00;1693679227
180313;1010;2005;26.32;1693680804
180571;1074;2009;41.18;1693680831
180830;1085;2004;35.14;1693686537
181090;1024;2006;100.00;1693690110
181351;1015;2006;77.08;1693691325
181613;1013;2004;0.00;1693695288
181876;1039;2007;87.10;1693699456
182140;1080;2005;13.16;1693704782
182405;1041;2005;100.00;1693708759
182671;1031;2001;91.67;1693711984
182938;1038;2010;100.00;1693715809
183206;1077;2005;10.53;1693721517
183475;1004;2007;100.00;1693721519
183745;1034;2004;18.92;1693721529
184016;1048;2006;47.92;1693721577
184288;1009;2004;100.00;1693723721
184561;1019;2010;95.45;1693728882
184835;1022;2006;56.25;1693730148
185110;1067;2007;87.10;1693735844
185386;1023;2004;35.14;1693737303
185663;1099;2006;62.50;1693741434
185941;1069;2009;100.00;1693744976
186220;1043;2003;100.00;1693745406
186500;1039;2004;70.27;1693746123
186781;1013;2002;76.19;1693748831
187063;1098;2008;40.00;1693752980
187346;1005;2010;100.00;1693758762
187630;1029;2004;100.00;1693762035
187915;1003;2006;66.67;1693762755
188201;1082;2006;100.00;1693764747
188488;1047;2002;85.71;1693767861
188776;1094;2005;100.00;1693770248
189065;1051;2004;51.35;1693772400
189355;1030;2010;45.45;1693776989
189646;1089;2009;26.47;1693780869
189938;1023;2004;100.00;1693785885
190231;1007;2004;56.76;1693789582
190525;1031;2001;100.00;1693794699
190820;1074;2010;90.91;1693799376
191116;1012;2003;100.00;1693800750
191413;1076;2006;12.50;1693803036
191711;1020;2006;100.00;1693804238
192010;1071;2002;28.57;1693804524
192310;1073;2009;47.06;1693807104
192611;1091;2008;100.00;1693807835
192913;1052;2003;12.00;1693809064
193216;1095;2002;100.00;1693810608
193520;1065;2009;55.88;1693816107
193825;1099;2006;77.08;1693820886
194131;1093;2010;77.27;1693825438
194438;1094;2005;0.00;1693825585
194746;1055;2007;48.39;1693830972
195055;1097;2002;100.00;1693831451
195365;1022;2001;100.00;1693832957
195676;1083;2007;100.00;1693837357
195988;1003;2008;15.56;1693840826
196301;1094;2008;100.00;1693841707
196615;1062;2010;27.27;1693846318
196930;1006;2003;88.00;1693850848
197246;1055;2005;63.16;1693851790
197563;1072;2009;0.00;1693855739
197881;1055;2001;31.25;1693856067
198200;1074;2006;100.00;1693859001
198520;1075;2003;16.00;1693859411
198841;1089;2004;83.78;1693859632
199163;1016;2003;60.00;1693864380
199486;1016;2004;67.57;1693866223
199810;1007;2001;54.17;1693868356
200135;1047;2003;20.00;1693871354
200461;1023;2006;97.92;1693873533
200788;1066;2001;79.17;1693873669
201116;1061;2004;91.89;1693875132
201445;1007;2005;50.00;1693879308
201775;1004;2002;61.90;1693879700
202106;1005;2002;100.00;1693883107
202438;1093;2002;80.95;1693887282
202771;1002;2005;2.63;1693891446
203105;1100;2006;29.17;1693891684
203440;1083;2001;2.08;1693892853
203776;1059;2003;100.00;1693895323
204113;1011;2001;60.42;1693896372
204451;1002;2003;100.00;1693901707
204790;1010;2008;22.22;1693903564
205130;1024;2007;58.06;1693905812
205471;1023;2001;27.08;1693908225
205813;1006;2005;15.79;1693908528
206156;1051;2008;100.00;1693908909
206500;1089;2010;100.00;1693914315
206845;1039;2002;9.52;1693917235
207191;1037;2001;33.33;1693917689
207538;1080;2006;100.00;1693918905
207886;1027;2004;24.32;1693920252
208235;1033;2010;81.82;1693921650
208585;1002;2010;22.73;1693927195
208936;1078;2009;79.41;1693932620
209288;1094;2004;72.97;1693937167
209641;1088;2009;94.12;1693938891
209995;1012;2001;100.00;1693944848
210350;1052;2004;97.30;1693945418
210706;1046;2003;100.00;1693946630
211063;1007;2008;46.67;1693947630
211421;1038;2001;100.00;1693950216
211780;1014;2007;61.29;1693956087
212140;1100;2004;100.00;1693959574
212501;1066;2008;100.00;1693960619
212863;1070;2003;64.00;1693963281
213226;1050;2002;0.00;1693967042
213590;1094;2003;68.00;1693970748
213955;1034;2001;87.50;1693973772
214321;1062;2006;100.00;1693979664
214688;1028;2005;100.00;1693981431
215056;1087;2008;22.22;1693983091
215425;1079;2009;11.76;1693986790
215795;1066;2005;94.74;1693991536
216166;1061;2002;100.00;1693992903
216538;1044;2007;100.00;1693995453
216911;1037;2008;100.00;1694001447
217285;1028;2010;18.18;1694006427
217660;1015;2004;100.00;1694007302
218036;1022;2002;100.00;1694010057
218413;1068;2010;100.00;1694014155
218791;1018;2005;18.42;1694014194
219170;1021;2005;36.84;1694017614
219550;1057;2003;56.00;1694021784
219931;1035;2002;100.00;1694025713
220313;1027;2003;12.00;1694031496
220696;1095;2002;95.24;1694035443
221080;1091;2009;76.47;1694037376
221465;1074;2009;38.24;1694037573
221851;1083;2001;100.00;1694041336
222238;1015;2004;35.14;1694044879
222626;1003;2009;55.88;1694046907
223015;1048;2009;5.88;1694052610
223405;1095;2001;100.00;1694055253
223796;1091;2008;84.44;1694059947
224188;1074;2006;39.58;1694061736
224581;1022;2004;100.00;1694063026
224975;1018;2009;38.24;1694068688
225370;1088;2005;18.42;1694071288
225766;1006;2002;66.67;1694072962
226163;1061;2004;83.78;1694077752
226561;1058;2003;84.00;1694083320
226960;1064;2002;38.10;1694086841
227360;1045;2009;11.76;1694092561
227761;1012;2004;48.65;1694097849
228163;1027;2001;100.00;1694101420
228566;1013;2007;41.94;1694103872
228970;1014;2007;100.00;1694107547
229375;1080;2008;22.22;1694108272
229781;1068;2004;43.24;1694109173
230188;1032;2007;70.97;1694112182
230596;1013;2006;47.92;1694116567
231005;1026;2005;81.58;1694118410
231415;1005;2006;66.67;1694120209
231826;1011;2005;10.53;1694122257
232238;1022;2004;5.41;1694123000
232651;1001;2006;100.00;1694125251
233065;1051;2001;60.42;1694129426
233480;1035;2006;47.92;1694131827
233896;1006;2005;63.16;1694135441
234313;1073;2001;100.00;1694138352
234731;1056;2005;71.05;1694143490
235150;1064;2010;100.00;1694146464
235570;1006;2002;100.00;1694150330
235991;1037;2008;57.78;1694154171
236413;1070;2006;79.17;1694159893
236836;1047;2004;35.14;1694164222
237260;1082;2007;100.00;1694167723
237685;1077;2003;76.00;1694172002
238111;1076;2008;42.22;1694176268
238538;1019;2001;16.67;1694180244
238966;1039;2001;100.00;1694181612
239395;1005;2003;100.00;1694186183
239825;1013;2005;97.37;1694189185
240256;1021;2002;23.81;1694193299
240688;1078;2009;91.18;1694195872
241121;1025;2006;100.00;1694198157
241555;1099;2002;28.57;1694198513
241990;1094;2009;100.00;1694199349
242426;1055;2007;83.87;1694201755
242863;1031;2002;100.00;1694207589
243301;1082;2003;36.00;1694210792
243740;1089;2008;93.33;1694212827
244180;1094;2006;45.83;1694216710
244621;1028;2005;100.00;1694219274
245063;1011;2008;37.78;1694223335
245506;1014;2001;72.92;1694223456
245950;1081;2010;50.00;1694228838
246395;1038;2010;68.18;1694233056
246841;1097;2009;44.12;1694237017
247288;1080;2001;43.75;1694241737
247736;1020;2002;100.00;1694246253
248185;1008;2004;59.46;1694250675
248635;1079;2004;21.62;1694253566
249086;1064;2008;20.00;1694255136
249538;1080;2002;100.00;1694258049
249991;1058;2008;77.78;1694261222
250445;1003;2006;25.00;1694261600
250900;1060;2006;100.00;1694263937
251356;1053;2001;50.00;1694265233
251813;1053;2005;63.16;1694266518
252271;1021;2006;70.83;1694270918
252730;1009;2010;100.00;1694272456
253190;1084;2003;8.00;1694277080
253651;1089;2002;38.10;1694280909
254113;1006;2001;25.00;1694283952
254576;1098;2003;28.00;1694285937
255040;1037;2009;97.06;1694286310
255505;1026;2001;100.00;1694292053
255971;1098;2008;95.56;1694292178
256438;1050;2010;68.18;1694292525
256906;1089;2002;100.00;1694297227
257375;1038;2001;27.08;1694300961
257845;1010;2006;66.67;1694304688
258316;1042;2007;93.55;1694307383
258788;1050;2007;77.42;1694310203
259261;1066;2007;80.65;1694310581
259735;1054;2003;92.00;1694312600
260210;1060;2007;3.23;1694316797
260686;1058;2009;8.82;1694317071
261163;1001;2006;35.42;1694319236
261641;1025;2009;88.24;1694319975
262120;1021;2009;26.47;1694320460
262600;1092;2006;97.92;1694321460
263081;1003;2006;22.92;1694322724
263563;1040;2007;9.68;1694328313
264046;1009;2009;88.24;1694330269
264530;1012;2005;15.79;1694330435
265015;1071;2008;35.56;1694331899
265501;1099;2003;100.00;1694333419
265988;1013;2001;83.33;1694337000
266476;1081;2004;21.62;1694341193
266965;1014;2005;92.11;1694343622
267455;1004;2007;9.68;1694348120
267946;1024;2010;100.00;1694349542
268438;1090;2007;64.52;1694350955
268931;1045;2006;64.58;1694355821
269425;1005;2003;68.00;1694356626
269920;1003;2004;37.84;1694361340
270416;1050;2009;47.06;1694361876
270913;1099;2007;100.00;1694366043
271411;1062;2001;100.00;1694367954
271910;1091;2004;100.00;1694369039
272410;1010;2003;20.00;1694374492
272911;1049;2010;90.91;1694378012
273413;1097;2006;35.42;1694383063
273916;1071;2009;58.82;1694385980
274420;1059;2010;22.73;1694391589
274925;1097;2008;97.78;1694391715
275431;1012;2008;73.33;1694396224
275938;1091;2005;100.00;1694397405
276446;1026;2003;100.00;1694401025
276955;1083;2005;63.16;1694406727
277465;1003;2010;45.45;1694410995
277976;1075;2009;100.00;1694416484
278488;1086;2002;61.90;1694417141
279001;1011;2003;64.00;1694418127
279515;1091;2002;80.95;1694419270
280030;1057;2008;26.67;1694422649
280546;1026;2004;67.57;1694425829
281063;1008;2008;88.89;1694428779
281581;1100;2009;100.00;1694434432
282100;1055;2003;4.00;1694439887
282620;1096;2007;6.45;1694445194
283141;1086;2010;4.55;1694445880
283663;1081;2004;100.00;1694449457
284186;1019;2009;32.35;1694450570
284710;1098;2009;100.00;1694450854
285235;1050;2007;74.19;1694453876
285761;1094;2006;100.00;1694456619
286288;1025;2009;50.00;1694461292
286816;1045;2005;73.68;1694466444
287345;1045;2003;100.00;1694470862
287875;1065;2010;100.00;1694475729
288406;1076;2008;62.22;1694477765
288938;1033;2004;100.00;1694479601
289471;1025;2001;100.00;1694484134
290005;1014;2007;6.45;1694485766
290540;1085;2007;93.55;1694488964
291076;1091;2008;84.44;1694489504
291613;1039;2007;16.13;1694492036
292151;1028;2003;84.00;1694496551
292690;1058;2006;100.00;1694500176
293230;1084;2004;100.00;1694504403
293771;1022;2007;93.55;1694507790
294313;1051;2006;97.92;1694513423
294856;1092;2009;100.00;1694518762
295400;1083;2009;38.24;1694524456
295945;1070;2002;38.10;1694528953
296491;1005;2007;58.06;1694534680
297038;1035;2007;35.48;1694536135
297586;1066;2004;51.35;1694538947
298135;1062;2008;100.00;1694543227
298685;1073;2010;40.91;1694544681
299236;1037;2004;100.00;1694550602
299788;1097;2001;8.33;1694555226
300341;1079;2001;100.00;1694559557
300895;1032;2001;41.67;1694562664
301450;1094;2005;100.00;1694566260
302006;1081;2009;17.65;1694567735
302563;1055;2005;65.79;1694570317
303121;1070;2001;100.00;1694574762
303680;1062;2008;13.33;1694578883
304240;1019;2009;100.00;1694581819
304801;1091;2001;100.00;1694584031
305363;1037;2007;96.77;1694586361
305926;1017;2010;13.64;1694591887
306490;1083;2006;100.00;1694592594
307055;1057;2005;100.00;1694596990
307621;1029;2003;88.00;1694597853
308188;1062;2007;48.39;1694601650
308756;1038;2007;74.19;1694607174
309325;1030;2009;100.00;1694613113
309895;1087;2002;0.00;1694613510
310466;1002;2008;51.11;1694615183
311038;1032;2008;100.00;1694615202
311611;1037;2003;0.00;1694620996
312185;1010;2001;79.17;1694623446
312760;1006;2010;54.55;1694628094
313336;1050;2005;89.47;1694632436
313913;1006;2006;35.42;1694633315
314491;1021;2006;100.00;1694633484
315070;1076;2004;45.95;1694637456
315650;1040;2007;87.10;1694640361
316231;1012;2009;100.00;1694644687
316813;1071;2003;100.00;1694646706
317396;1030;2006;4.17;1694647617
317980;1096;2004;72.97;1694649145
318565;1096;2005;42.11;1694649201
319151;1046;2001;64.58;1694653708
319738;1061;2009;100.00;1694657332
320326;1074;2010;100.00;1694659583
320915;1060;2003;64.00;1694661311
321505;1004;2004;54.05;1694667165
322096;1044;2002;57.14;1694670199
322688;1023;2008;77.78;1694674958
323281;1097;2004;100.00;1694680194
323875;1065;2007;100.00;1694681887
324470;1027;2009;79.41;1694687870
325066;1015;2004;89.19;1694692941
325663;1045;2005;68.42;1694696633
326261;1018;2003;100.00;1694701033
326860;1056;2001;100.00;1694703257
327460;1022;2007;58.06;1694703518
328061;1077;2001;100.00;1694704905
328663;1036;2006;97.92;1694710512
329266;1073;2003;48.00;1694714998
329870;1098;2007;74.19;1694718116
330475;1096;2004;5.41;1694723572
331081;1091;2004;35.14;1694728761
331688;1008;2009;50.00;1694732531
332296;1083;2008;77.78;1694737808
332905;1007;2005;100.00;1694742423
333515;1024;2010;100.00;1694748084
334126;1086;2008;53.33;1694752324
334738;1081;2005;100.00;1694757064
335351;1098;2004;100.00;1694761207
335965;1083;2008;100.00;1694764517
336580;1041;2003;100.00;1694765856
337196;1078;2004;21.62;1694770444
337813;1034;2009;38.24;1694774481
338431;1037;2003;32.00;1694776045
339050;1065;2005;100.00;1694781913
339670;1074;2007;100.00;1694785417
340291;1009;2004;100.00;1694791014
340913;1096;2003;56.00;1694793167
341536;1014;2004;75.68;1694798980
342160;1089;2009;76.47;1694803237
342785;1027;2008;51.11;1694803544
343411;1038;2001;52.08;1694805415
344038;1028;2004;54.05;1694809440
344666;1044;2008;44.44;1694814253
345295;1097;2001;20.83;1694815714
345925;1018;2008;31.11;1694820652
346556;1059;2010;22.73;1694824217
347188;1065;2005;100.00;1694826463
347821;1037;2010;13.64;1694826692
348455;1063;2008;100.00;1694830116
349090;1037;2009;100.00;1694830523
349726;1046;2006;100.00;1694831035
350363;1087;2005;100.00;1694831729
