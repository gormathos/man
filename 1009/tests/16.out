191
25.08.2023 09:01:03
25.08.2023 11:48:06
25.08.2023 17:57:15
25.08.2023 19:24:53
26.08.2023 04:43:46
26.08.2023 09:43:45
26.08.2023 16:03:16
26.08.2023 16:55:39
26.08.2023 22:38:12
27.08.2023 04:28:54
27.08.2023 07:43:43
27.08.2023 15:11:15
27.08.2023 21:47:16
27.08.2023 23:23:32
28.08.2023 02:52:24
28.08.2023 04:13:42
28.08.2023 04:40:53
28.08.2023 04:48:20
28.08.2023 07:25:10
28.08.2023 07:34:06
28.08.2023 13:58:00
28.08.2023 14:07:36
28.08.2023 16:48:02
28.08.2023 18:35:13
28.08.2023 19:02:11
28.08.2023 21:33:24
28.08.2023 23:05:51
29.08.2023 01:14:47
29.08.2023 04:27:01
29.08.2023 06:11:39
29.08.2023 12:05:38
29.08.2023 14:24:38
29.08.2023 16:53:00
29.08.2023 17:22:48
29.08.2023 17:42:26
29.08.2023 19:16:50
29.08.2023 20:31:04
30.08.2023 00:07:13
30.08.2023 00:45:26
30.08.2023 02:41:06
30.08.2023 06:06:18
30.08.2023 07:20:10
30.08.2023 09:14:31
30.08.2023 10:29:56
30.08.2023 12:30:09
30.08.2023 14:12:51
30.08.2023 14:39:12
30.08.2023 14:40:46
30.08.2023 14:46:57
30.08.2023 17:10:13
30.08.2023 20:23:24
31.08.2023 09:19:23
31.08.2023 11:11:28
31.08.2023 15:50:17
31.08.2023 17:16:03
31.08.2023 18:38:42
31.08.2023 20:05:33
31.08.2023 23:33:58
01.09.2023 00:52:03
01.09.2023 00:55:42
01.09.2023 02:26:37
01.09.2023 03:43:06
01.09.2023 05:26:02
01.09.2023 10:27:47
01.09.2023 15:02:22
01.09.2023 17:14:10
01.09.2023 17:17:13
01.09.2023 19:33:12
01.09.2023 22:55:05
02.09.2023 02:09:36
02.09.2023 03:49:24
02.09.2023 15:05:18
02.09.2023 18:27:07
02.09.2023 21:28:30
03.09.2023 02:39:19
03.09.2023 04:36:49
03.09.2023 06:11:59
03.09.2023 06:48:41
03.09.2023 12:42:56
03.09.2023 12:50:06
03.09.2023 16:32:42
03.09.2023 17:27:15
03.09.2023 18:12:27
03.09.2023 19:44:08
04.09.2023 00:04:45
04.09.2023 02:31:39
04.09.2023 04:12:30
04.09.2023 05:10:38
04.09.2023 06:10:35
04.09.2023 06:56:48
04.09.2023 12:44:11
04.09.2023 13:09:17
04.09.2023 14:22:37
04.09.2023 15:35:07
04.09.2023 20:23:21
05.09.2023 03:05:07
05.09.2023 06:28:43
05.09.2023 08:15:07
05.09.2023 10:15:09
05.09.2023 11:45:15
05.09.2023 13:01:45
05.09.2023 20:14:08
05.09.2023 20:43:50
05.09.2023 21:43:36
06.09.2023 00:19:34
06.09.2023 00:36:59
06.09.2023 05:54:24
06.09.2023 06:23:51
06.09.2023 09:35:03
06.09.2023 10:17:33
06.09.2023 11:57:27
06.09.2023 13:35:02
06.09.2023 14:20:57
06.09.2023 15:29:15
06.09.2023 18:41:53
06.09.2023 23:02:16
07.09.2023 02:54:13
07.09.2023 05:03:46
07.09.2023 15:43:40
07.09.2023 17:25:47
07.09.2023 22:20:51
08.09.2023 01:59:12
08.09.2023 04:14:24
08.09.2023 05:18:50
08.09.2023 10:08:43
08.09.2023 14:00:12
08.09.2023 15:16:23
08.09.2023 18:35:57
08.09.2023 18:55:49
08.09.2023 21:13:09
09.09.2023 00:27:54
09.09.2023 07:57:33
09.09.2023 11:14:09
09.09.2023 12:52:17
09.09.2023 15:14:16
09.09.2023 20:40:53
09.09.2023 22:07:07
10.09.2023 08:10:19
10.09.2023 12:39:02
10.09.2023 17:14:03
10.09.2023 17:45:54
10.09.2023 18:03:59
11.09.2023 01:56:45
11.09.2023 02:57:05
11.09.2023 07:14:44
11.09.2023 12:13:52
11.09.2023 16:24:17
11.09.2023 16:47:34
11.09.2023 18:23:39
11.09.2023 22:21:02
11.09.2023 23:42:09
12.09.2023 00:46:41
12.09.2023 02:02:14
12.09.2023 06:29:36
12.09.2023 07:40:03
12.09.2023 11:39:22
12.09.2023 18:27:07
12.09.2023 20:30:02
12.09.2023 22:59:17
13.09.2023 00:51:00
13.09.2023 03:12:42
13.09.2023 05:10:19
13.09.2023 05:47:11
13.09.2023 08:09:54
13.09.2023 09:23:10
13.09.2023 13:51:53
13.09.2023 14:26:42
13.09.2023 19:31:24
13.09.2023 22:38:07
13.09.2023 23:11:46
14.09.2023 02:08:52
14.09.2023 02:46:23
14.09.2023 08:29:54
14.09.2023 08:58:07
14.09.2023 14:17:13
14.09.2023 14:54:17
14.09.2023 15:21:45
15.09.2023 01:47:03
15.09.2023 03:21:24
15.09.2023 05:51:04
15.09.2023 07:00:07
15.09.2023 07:55:17
15.09.2023 08:17:36
15.09.2023 12:45:13
15.09.2023 13:43:37
15.09.2023 15:16:54
16.09.2023 01:07:43
16.09.2023 02:08:36
16.09.2023 02:15:23
16.09.2023 02:23:55
16.09.2023 02:35:29
